node/parallel/src/client.rs

```diff
@@ -176,7 +180,9 @@ impl ClientHandle for Client {
 impl sc_client_api::UsageProvider<Block> for Client {
     fn usage_info(&self) -> sc_client_api::ClientInfo<Block> {
         match self {
+            #[cfg(feature = "with-parallel-runtime")]
```

Comment:
I remember we discussed that won't increase code complexity with these features for build speedup.

Reply:
looks like the only difference is `WithdrawAmount`  config
![Uploading image.png…]()

Reply:
I reverted that commit for now